fswatch/src/fswatch.cpp

```diff
@@ -27,14 +27,14 @@
 #include <vector>
 #include <array>
 #include <map>
-#include "libfswatch/c++/path_utils.hpp"
-#include "libfswatch/c++/event.hpp"
-#include "libfswatch/c++/monitor.hpp"
-#include "libfswatch/c++/monitor_factory.hpp"
-#include "libfswatch/c/error.h"
-#include "libfswatch/c/libfswatch.h"
-#include "libfswatch/c/libfswatch_log.h"
-#include "libfswatch/c++/libfswatch_exception.hpp"
+#include "path_utils.hpp"
+#include "event.hpp"
+#include "monitor.hpp"
+#include "monitor_factory.hpp"
+#include "fsw_error.h"
```

Comment:
If we rename files and change the include paths, we should keep the build system working, and autotools are now broken